core-service/src/main/java/com/crmfoodestablishment/coreservice/entity/Dish.java

```diff
@@ -0,0 +1,29 @@
+package com.crmfoodestablishment.coreservice.entity;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Entity
+public class Dish {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id", nullable = false, unique = true)
+    private Integer id;
+
+    @Column(name = "price", nullable = false)
+    private Double price;
```

Comment:
double or bigDecimal everywhere ?

Reply:
I think BigDecimal will be more suitable for price